### PR TITLE
Improve contributing docs

### PR DESCRIPTION
**Pull request by AlekSi**

### CODE REVIEW

**Review by w84thesun — APPROVED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
 # Checklist
 
+* [ ] I read [CONTRIBUTING.md](../CONTRIBUTING.md).
```
Comment (seeforschauer) [inline .github/pull_request_template.md:3]: I would suggest adding here some links / highlites from contributing guidelines also.

Comment (rumyantseva) [inline .github/pull_request_template.md:3]:
```suggestion
* [ ] I've read [CONTRIBUTING.md](../CONTRIBUTING.md). I follow the guidelines provided there.
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,6 +1,5 @@
...
 * [ ] Tests are added for new functionality or fixed bugs.
```
Comment (rumyantseva) [inline .github/pull_request_template.md:4]:
```suggestion
* [ ] I added tests for the new functionality or fixed bugs.
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,6 +1,5 @@
...
 * [ ] `task all` passes.
```
Comment (rumyantseva) [inline .github/pull_request_template.md:5]:
```suggestion
* [ ] I called `task all` locally, and it passed.
```

**Review by rumyantseva — COMMENTED** [review]
Added a couple of wording suggestions.

File: .github/pull_request_template.md (hunk 2 above)
Comment (AlekSi) [reply .github/pull_request_template.md:4]: My idea was to use exactly the same wording there and in CONTRIBUTING.md to make it easier to copy&paste

Comment (rumyantseva) [reply .github/pull_request_template.md:4]: @AlekSi well, my concern is that for the first sentence you used active voice, and for the rest of the items - passive voice. So, there wasn't enough consistency there. Also, I believe, active voice is recommended when you implement a "call for action". Anyways, I leave it up to you.

**Review by w84thesun — CHANGES_REQUESTED** [review]
I like the idea to reword the checklist to active voice.

**Pull request closed without merging**